### ISSUE TITLE
[FEATURE_REQUEST] Allow using different faviconApi for each items

### ISSUE THREAD

**Issue by phongngthanh** [Lissy93/dashy#196]
### Is your feature request related to a problem? If so, please describe.

Some of my internet-facing site have basic-auth, so the faviconkit API cannot get the icon for them. For example:
```
appConfig:
  faviconApi: faviconkit
  statusCheck: true
  theme: Material
sections:
  - name: Media management
    items:
# This service has basic-auth
      - icon: favicon
        statusCheckAllowInsecure: true
        statusCheckUrl: http://radarr:7878
        title: Radarr
        url: https://radarr.{{ server_domain }}
# This service doesn't have basic-auth
      - icon: favicon
        title: Nextcloud
        url: https://nextcloud.{{ server_domain }}
```
Using faviconkit will not return a favicon for the site with basic-auth.
![image](https://user-images.githubusercontent.com/67867662/131777570-bf3d823b-e6ac-4c10-beef-ae0e66262d13.png)
If I change the faviconApi to local, I'm not sure how it works, but the icon appears normally. However, other services without basic-auth doesn't have icons anymore.
![image](https://user-images.githubusercontent.com/67867662/131777525-758d833a-b558-4ee8-a294-b4a2c926bc7d.png)

### Describe the solution you'd like

Allow using different faviconApi for each items entry. Something like this:
```
appConfig:
  statusCheck: true
  theme: Material
sections:
  - name: Media management
    items:
# This service has basic-auth
      - icon: favicon
        faviconApi: local
        statusCheckAllowInsecure: true
        statusCheckUrl: http://radarr:7878
        title: Radarr
        url: https://radarr.{{ server_domain }}
# This service doesn't have basic-auth
      - icon: favicon
        faviconApi: faviconkit
        title: Nextcloud
        url: https://nextcloud.{{ server_domain }}
```

### Priority

Low (Nice-to-have)

**Comment by Lissy93** [comment]
Thanks for the suggestion, I can certainly implement this, should be a quick task. Will do it this weekend :)

**Comment by Lissy93** [comment]
Hey @phongngthanh - This is implemented in b756e65  of #201 released in V 1.7.1. 

You can now use a different favicon API for certain items, just set `icon: favicon-[api]`, e.g. `favicon-clearbit`

Hope that helps :)
